app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<section class="d-flex-col user-container center">
+  <h2>Sign up</h2>
```

Comment:
- [Optional] @AndreaM2429, you didn't modify forms to match the registration and login forms as pointed in the requirements. Let's kindly modify these inputs fields.

| Your Design | Wireframe |
| --- | --- |  
| ![Screenshot from 2023-10-05 04-23-51](https://github.com/AndreaM2429/BlogApp/assets/63087491/4f8be337-d5a3-44d9-9721-c5078cb1ee97) | ![Screenshot from 2023-10-05 04-24-20](https://github.com/AndreaM2429/BlogApp/assets/63087491/491aefb2-b31c-474b-ae85-5b0d2361b9ef) |